types/node-red__editor-client/node-red__editor-client-tests.ts

```diff
@@ -0,0 +1,370 @@
+// tslint:disable:space-before-function-paren
+
```

Comment:
```suggestion
```

Reply:
This can't be removed, because the parens are added by prettier. Try to update the `function ()` -> `function()` locally and run `npm run prettier types/node-red__editor-client/node-red__editor-client-tests.ts`.

Recent changes in Prettier 2.0:

> Previously, a space would be added after the function keyword in function declarations, but not in function expressions. Now, for consistency, a space is always added after the function keyword. The only exception is generator declarations where function* is treated as a whole word.

Source: https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot